src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -2168,6 +2168,122 @@ public void test_merge_strategy_correct_fail() throws Exception {
         }
     }
 
+    @Bug(12402)
```

Comment:
Thanks for letting me know. I was just copying what I saw elsewhere in that file. Would you like me to change all the @Bug in this project or just this one? Looks like a clean up activity that should be on its own PR isn't it?